pkg/workloads/console/integration/integration_test.go

```diff
@@ -188,15 +192,38 @@ var _ = Describe("Console", func() {
 			By("Expect job was created")
 			job := &batchv1.Job{}
 			identifier, _ := client.ObjectKeyFromObject(csl)
+			identifier.Name += "-console"
 			err := mgr.GetClient().Get(context.TODO(), identifier, job)
 
 			Expect(err).NotTo(HaveOccurred(), "failed to find associated Job for Console")
+
+			By("Expect job and pod labels are correctly populated from the console template and console")
+			Expect(
+				job.ObjectMeta.Labels["user"]).To(Equal("system-unsecured"))
+			Expect(
+				job.ObjectMeta.Labels["console-name"]).To(Equal(csl.ObjectMeta.Name))
+			Expect(
+				job.ObjectMeta.Labels["repo"]).To(Equal(consoleTemplate.ObjectMeta.Labels["repo"]),
+				"job's has same labels as the console template (preference over console labels)",
+			)
+			Expect(
+				job.ObjectMeta.Labels["other-label"]).To(Equal(csl.ObjectMeta.Labels["other-label"]),
+				"job's has same labels as the console",
+			)
+			Expect(
+				job.Spec.Template.Labels["job-name"]).To(Equal(job.ObjectMeta.Name),
```

Comment:
I'm a bit confused where this is coming from - is the apiserver automatically setting it when we create the job object?

If that's the case, then we probably shouldn't be testing it here?

Reply:
I agreed that now it doesn't make have this here.